### PR TITLE
Update main.go

### PR DESCRIPTION
**Pull request by msteggink**
Made acknowledgements configurable.

### CODE REVIEW

**Comment by msteggink** [comment]
My first go, please review. Thanks!

File: main.go [hunk 1]
```diff
@@ -17,6 +17,7 @@ var (
 	uri          = flag.String("uri", "amqp://guest:guest@localhost:5672/", "AMQP URI")
 	insecure_tls = flag.Bool("insecure-tls", false, "Insecure TLS mode: don't check certificates")
 	queue        = flag.String("queue", "", "AMQP queue name")
+        ack          = flag.Bool("ack", false, "Acknowledge messages")
```
Comment (dubek) [inline main.go]: I think this line is indented with spaces instead of tabs which are recommended by Go style. Please use `go fmt` to format the code.

**Review by dubek — CHANGES_REQUESTED** [review]
Thanks for adding this!

The code change itself looks good except the small formatting issue.

Please add a test to `main_test.go` testing this new feature. For example it can populate a queue with 10 messages, dump 3 messages, then dump 10 more (and expect exactly 7 to be dumped).

Please add a small paragraph to `README.md` explaining the new switch. Using this switch will actually *drain* the queue from the N messages, so in that sense it is a dangerous operation and should be explained as such.

**Comment by msteggink** [comment]
The test has been added, please let me know if this is acceptable.

**Comment by dubek** [comment]
Looks good! Merging.

**Comment by dubek** [comment]
One last comment (for next time): please make the commit message more descriptive: instead of "update main.go" I'd write something like "Support -ack option for removing messages from queue".

**Comment by dubek** [comment]
This is now part of release v0.4 available via the [releases page](https://github.com/dubek/rabbitmq-dump-queue/releases).

**Pull request merged**